lib/gulp-tasks/patterns-import.js

```diff
@@ -0,0 +1,72 @@
+/**
+ *  @fileOverview
+ * Uses Gulpjs to import html patterns
+ *
+ *  @author       Scott Nath
+ *
+ *  @requires     NPM:gulp
+ *  @requires     NPM:lodash.merge
+ *  @requires     NPM:pattern-importer
+ */
+
+var gulp = require('gulp'),
+  merge = require('lodash.merge'),
+  utility = require('../utility');
+
+/**
+ * Function to get default options for an implementation of patternlab-import
+ * @returns {Object}  options  an object of default patternlab-import options
+ */
+var getDefaultOptions = function () {
```

Comment:
@scottnath Yes, you are correct, and so long as this is Node, this is safe. However, I don't think it's a good idea to use the stricter practice in the browser and a less strict practice in Node. You do no harm declaring a function from within a function in both cases.